integration-tests/api/__tests__/categories/category.ts

```diff
@@ -0,0 +1,85 @@
+import path from "path"
+import { Category } from "@medusajs/medusa"
+import { initDb, useDb } from "../../../helpers/use-db"
+
+describe("Categories > Materialized Paths", () => {
+  let dbConnection
+
+  beforeAll(async () => {
+    const cwd = path.resolve(path.join(__dirname, "..", ".."))
+    dbConnection = await initDb({ cwd })
+  })
+
+  afterAll(async () => {
+    const db = useDb()
+    await db.shutdown()
+  })
+
+  afterEach(async () => {
+    const db = useDb()
+    await db.teardown()
+  })
+
+  it("can fetch ancestors, descendents and root categories", async () => {
+    const categoryRepository = dbConnection.getTreeRepository(Category)
+
+    const a1 = new Category()
```

Comment:
suggestion(non blocking): `const a1 = categoryRepository.create({ name: "", handle: "" })` same for the others